src/App.tsx

```diff
@@ -1,7 +1,35 @@
-import React from 'react';
+/* eslint-disable no-console */
+import React, { Key } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: Key | null;
+};
+
+export class App extends React.Component<{}, State> {
+  state: State = {
```

Comment:
It's typed here <{}, State>
```suggestion
  state = {
```